cmake/modules/BuildFIO.cmake

```diff
@@ -0,0 +1,15 @@
+function(build_fio)
+  # we use an external project and copy the sources to bin directory to ensure
+  # that object files are built outside of the source tree.
+  include(ExternalProject)
+  ExternalProject_Add(fio_ext
+    DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/src/
+    GIT_REPOSITORY "https://github.com/axboe/fio.git"
+    GIT_TAG "e09d68c8da4ab91397490577454de928106651f5"
```

Comment:
is there anything special about this commit? why not target the master branch?

Reply:
After considering these two options: lock the commit or bind to arbitrary selected one I choose the latter -  it will prevent from occasional build failures after some changes in fio code base.

Reply:
I meann attaching to master instead of  "lock the commit" above.

Reply:
Replace specific commit with a v3.5 tag though.